src/trace-writer.js

```diff
@@ -186,10 +202,15 @@ TraceWriter.prototype.writeSpan = function(spanData) {
 TraceWriter.prototype.queueTrace_ = function(trace) {
   var that = this;
 
+  if (!that.isActive) {
+    return;
```

Comment:
As discussed in person, we will move back toward an unpatching/`stop` approach but `stop` will still be removed from the public interface.